kokoro/linux/dockerfile/test/java_stretch/Dockerfile

```diff
@@ -29,3 +29,10 @@ RUN apt-get install -y \
   # -- For java_linkage_monitor \
   openjdk-8-jdk \
   && apt-get clean
+
+# Python dependencies
+RUN apt-get install -y \
```

Comment:
You could merge the apt commands together, if you want. That would make the container slightly faster to build.